VTTiny/Components/Animator/AnimatorCharacter.Editor.cs

```diff
@@ -20,12 +24,70 @@ public void DrawEditorGUI(AssetDatabase assetDatabase)
 
             if (EditorGUI.AssetDropdown("Idle blinking", assetDatabase, IdleBlink, out Texture newBlinking))
                 IdleBlink = newBlinking;
-
-            if (EditorGUI.AssetDropdown("Speaking", assetDatabase, Speaking, out Texture newSpeaking))
-                Speaking = newSpeaking;
-
+            
             if (EditorGUI.AssetDropdown("Speaking blinking", assetDatabase, SpeakingBlink, out Texture newSpeakingBlink))
                 SpeakingBlink = newSpeakingBlink;
+            
+            //Continuously add Speaking textures
+
+            Speaking ??= new List<Texture>()
+            {
+                new Texture()
+            };
+
+            if (Speaking.Count() >= 0)
+            {
+                //remove all textures that are null
+                //var empty = Speaking.Where(x=>x == default(Texture)).ToList().Count() -1;
+
+                //Check for null textures
+                // for (int i = 0; i < empty; i++)
+                // {
+                //     Speaking.Remove(default(Texture));
+                // }
+
+                if (Speaking.Any() && Speaking.Last() != default)
+                {
+                    var i = Speaking.Last();
+                    if (i.Id != default)
+                    {
+                        //Add button to add new speaking texture
+                        Speaking.Add(new Texture());
+                        
+                    }
+
+                }
+
+                if (!Speaking.Any())
+                {
+                    Speaking.Add(new() );
+                }
+                for (int i = 0; i < Speaking.Count(); i++)
+                {
+                    if (EditorGUI.AssetDropdown("Speaking " + i, assetDatabase, Speaking[i], out Texture newSpeaking))
+                        Speaking[i] = newSpeaking;
+                    
+                }
+                // //if the last texture is not null, add a new one
+                // if (Speaking.Last() != default(Texture))
+                // {
+                //     Console.WriteLine("Adding new speaking texture");
+                //     Speaking.Add(new Texture());
+                // }
+            }
+            else
+            {
+                if (Speaking.Count() == 0)
+                {
+                    Speaking.Add(new Texture());
```

Comment:
oh no, this is not for adding a new texture, this will make a new place holder to put a new texture in, this is so you can have 9999 different kinds of mouth movements, this is just there in case there is 0 in the list, it'll make a temporary one